src/hooks/useContract.ts

```diff
@@ -137,8 +137,8 @@ export function useStableSwapContract(
   const { chainId, library } = useActiveWeb3React()
 
   const pool = poolName == null ? null : STABLESWAP_POOLS[ChainId.AURORA][poolName]
-  const metaPool = useStableSwapMetaPoolDeposit(pool?.address, withSignerIfPossible)
-  const metaPoolUnwrappedTokens = useStableSwapMetaPoolDeposit(pool?.metaSwapAddresses, withSignerIfPossible)
+  const metaPoolUnwrappedTokens = useStableSwapMetaPoolDeposit(pool?.address, withSignerIfPossible)
```

Comment:
Swapped these around to align with the swap around in the stableswap pool constants metaSwapAddresses and address key 

@specialist0x